lms/envs/common.py

```diff
@@ -773,7 +773,6 @@
 CMS_BASE = 'localhost:8001'
 
 # Site info
-SITE_ID = 1
```

Comment:
@symbolist I'm not sure we should remove that usage so much as protect against the scenario where the value is not available in settings -- for example:

`if hasattr(settings, "SITE_ID") and site_id != str(settings.SITE_ID):`

Thoughts?